src/org/spdx/rdfparser/model/RdfModelObject.java

```diff
@@ -436,6 +438,28 @@ protected void setPropertyValue(String nameSpace, String propertyName,
 		}
 	}
 	
+	protected void setPropertyValue(String nameSpace, String propertyName,
```

Comment:
This is OK - but I would prefer to translate the JSONArray to a Java array of values outside of the RdfModelObject to reduce the dependency graph for the RdfModelObject.  If the current caller(s) of this method translated into a standard Java object or primitive type, it would be a cleaner design IMHO.